DocTest.hs

```diff
@@ -1,6 +1,8 @@
 import Test.DocTest
-main = doctest [ "-isrc"
-               , "-XOverloadedStrings"
+main = doctest [ "-XOverloadedStrings"
+               , "-XCPP"
+               , "-XLambdaCase"
+               , "lib/Data/Aeson/Combinators/Compat.hs"
```

Comment:
Doctest doesn't work for me, due to CPP, not sure whether I broke it or it was broken before